### DISCUSSION TITLE
want to work on a new issue ? Read this

### DISCUSSION THREAD

**Discussion by codewarnab** [Innovateninjas/Paws-frontend#340] [Announcements]
when we assign a new issue to you if this issue does not contain any detailed description/approach  of the work  please try to come up with a detailed roadmap/approach  what you are going to do so this will less the chances of the back and forth during the code review in the pr 
example : in this [pr](https://github.com/Innovateninjas/Paws-frontend/issues/243)

![image](https://github.com/Innovateninjas/Paws-frontend/assets/94959855/e30d6ec7-0f70-4cb7-ae2a-ae5a17d8cbd1)

**Comment by siddhanth4** [comment]
Hi Arnab, I am assigned as Mentor for this project, So how can I send you the Maintainer Role request ?
